blog/poc3.md

```diff
@@ -1,120 +1,64 @@
+The POC3 code is now merged into the official [go-ethereum repository's master branch](https://github.com/ethereum/go-ethereum).
 
-Swarm is a service that provides APIs to upload and download content and through a url-based addressing offers *virtual hosting of websites* and decentralised applications (dapps) without webservers, using decentralised peer-to-peer distributed infrastructure.
-When content is uploaded to Swarm it is chopped up into pieces called *chunks*.
-Each chunk is identified with a unique address deterministically derived from its content (using the chunk hash).
-The hash addresses of data chunks are themselves packaged into a chunk which in turn has its own hash. In this way the content gets mapped into a [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree), the root hash of which, called the Swarm hash, is used as a *reference* to a file. When this reference then is used to retrieve the file, the client can recursively unpack the tree by fetching the individual chunks and assemble the original content. This hierarchical merklised Swarm hash construct offers *integrity protected random access* into large files;  permitting things like skipping to parts of a video or looking up a key in a database file.
 
-Most importantly, this means that, on the network layer, Swarm only needs to take care of storing and retrieving chunks.
-Swarm is a peer-to-peer network of nodes running the Swarm client. The network uses Ethereum's devp2p/rlpx networking layer to run the bzz suite of protocols. Nodes are identified with a Swarm address deterministically derived from the node's Ethereum account public key, which can place them in the same address space as the chunks themselves, so we can define proximity between nodes and chunks.
-The semi-permanent TCP connections between peers define an overlay network where messages can be relayed from node to node.
-The Swarm network implements a *distributed preimage archive (DPA)*, which is a  type of content addressed distributed chunk store in which nodes closest to the address of a chunk do not only serve information about the content but actually host the data. This DPA is the underlying distributed component of the Swarm service. When you upload content, the chunks need to reach the nodes that are closest to it (a process called *syncing*), so that when you retrieve chunks, nodes can send requests relayed towards the chunk's address via ever closer peers.
+It has been a year and a half since the first release of the POC2 series was deployed and [the Swarm project launched its public alpha network](https://blog.ethereum.org/2016/12/15/swarm-alpha-public-pilot-basics-swarm/).
+Two [Swarm summits](https://swarm-gateways.net/bzz:/swarm-orange-summit.eth), two [orange papers](http://swarm-guide.readthedocs.io/en/latest/resources.html#orange-papers) and [forty thousand lines of code](https://github.com/ethereum/go-ethereum/pull/17041) later, it is time to give an update.
```

Comment:
I know i rwote it but '[after a bloody long and eventful time] it is time to give an update' sounds like we were silent for too long...

Reply:
it is time to give an update -> it is time to take stock?